pkg/apis/core/validation/project_test.go

```diff
@@ -163,30 +163,117 @@ var _ = Describe("Project Validation Tests", func() {
 					"Detail":   Equal("must start with garden-"),
 				}))),
 			),
+			Entry("should forbid Project with an invalid namespace name",
+				"garden-foo%bar",
+				ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
+					"Type":     Equal(field.ErrorTypeInvalid),
+					"Field":    Equal("spec.namespace"),
+					"BadValue": Equal("garden-foo%bar"),
+					"Detail":   Equal(`a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')`),
+				}))),
+			),
 			Entry("should allow Project with namespace garden",
 				"garden",
 				BeEmpty(),
 			),
 		)
 
-		It("should forbid Project specification with empty or invalid key for description", func() {
-			project.Spec.Description = ptr.To("")
+		DescribeTable("Project spec.description",
+			func(description *string, matcher gomegatypes.GomegaMatcher) {
+				project.Spec.Description = description
 
-			errorList := ValidateProject(project)
+				errorList := ValidateProject(project)
 
-			Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
-				"Type":  Equal(field.ErrorTypeRequired),
+				Expect(errorList).To(matcher)
+			},
+			Entry("should allow no description",
```

Comment:
_nit:_ The description sounds to me as if no description is allowed. What do you think about the following?
```suggestion
			Entry("should allow unset description",
```